bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/property/ContractPropertySection.java

```diff
@@ -252,7 +253,7 @@ public void openAddInputWizardDialog() {
                         fieldToContractInputMappingOperationBuilder,
                         fieldToContractInputMappingExpressionBuilder,
                         BonitaStudioPreferencesPlugin.getDefault().getPreferenceStore(),
-                        sharedImages),
+                        sharedImages, new ContractInputGenerationInfoDialog()),
```

Comment:
I don't understand why you are passing  the Dialog in parameter. Can you explain please?